planning/behavior_path_planner/config/lane_change/lane_change.param.yaml

```diff
@@ -53,6 +53,7 @@
       abort_delta_time: 1.0                      # [s]
       aborting_time: 5.0                         # [s]
       abort_max_lateral_jerk: 1000.0             # [m/s3]
+      abort_exec_lateral_threshold: 0.0
```

Comment:
@zulfaqar-azmi-t4 @rej55 Any idea for a good naming?

Reply:
How is `overhang_threshold_for_continue_lane_change` ?

Reply:
or, `overhang_margin_for_continue_lane_change`

Reply:
changed to `lane_change.cancel.overhang_tolerance`